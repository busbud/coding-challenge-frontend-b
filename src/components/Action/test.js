import React from 'react';
import renderer from 'react-test-renderer';

import Component from './';

describe('Component', () => {
	it('should render button', () => {
		const wrapper = renderer.create(<Component />).toJSON();
		expect(wrapper).toMatchSnapshot();
	});

	it('should render link', () => {
		const wrapper = renderer.create(<Component classnames="link" />).toJSON();
		expect(wrapper).toMatchSnapshot();
	});
});
