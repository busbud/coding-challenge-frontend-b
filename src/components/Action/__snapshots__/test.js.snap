// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should render button 1`] = `
<button
  className="button"
  onClick={undefined}
/>
`;

exports[`Component should render link 1`] = `
<a
  className="link"
  onClick={undefined}
/>
`;
